debug.log

```diff
@@ -0,0 +1,80 @@
+# Logfile created on 2016-09-05 13:26:07 +1000 by logger.rb/47272
```

Comment:
I think once you've added it to git it ignores the .gitignore directive so you need to delete it from git. Then in future it will be ignore.